.reek

```diff
@@ -28,6 +28,7 @@ UnusedPrivateMethod:
     - ActiveJob::Logging::LogSubscriber
     - Users::PhoneConfirmationController
 UtilityFunction:
+  public_methods_only: true
```

Comment:
I suppose in theory this means we should be using class methods.. but then they can't be private... so I guess I agree with you.